Makefile

```diff
@@ -29,7 +29,7 @@ install-base-pip-packages:
 
 .PHONY: install-detectron2
 install-detectron2:
-	pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@78d5b4f335005091fe0364ce4775d711ec93566e"
+	pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@e2ce8dc1ab097891395d324abaffe9cf298503d1#egg=detectron2"
```

Comment:
LGTM
Just curiosity: there is any reason to choose that specific commit of detectron2 repo?

Reply:
I copied it from what's in the `Makefile` of the `unstructured` repo, and if I had to guess why it was chosen, I'd say it was probably the most recent version with passing tests at the time it was chosen a couple of weeks ago. Let me know if there's a concern about using this version vs the one you had.

Reply:
Yeah, I think the same, no problem.